src/nodes/dir.rs

```diff
@@ -247,6 +247,21 @@ impl Dir {
         }
         Dir::new_empty(ids.next(), Some(self), now)
     }
+
+    /// Same as `getattr` but with the node already locked.
+    fn getattr_unlocked(inode: u64, state: &mut MutableDir) -> NodeResult<fuse::FileAttr> {
```

Comment:
These kinds of functions are usually suffixed with `_locked`, aren't they?

Reply:
Yes. Dunno what I was thinking. Fixed the name.